apiserver/errors/errors.go

```diff
@@ -34,6 +34,12 @@ var (
 	ErrActionNotAvailable = errors.New("action no longer available")
 )
 
+// ErrTryAgain reports whether the cause
+// of the error is an ErrTryAgain.
+func IsErrTryAgain(err error) bool {
```

Comment:
There's a PR https://github.com/juju/juju/pull/13892 which converts some of these helpers to errors.Is() so we'll need to be sure to check to see if we need to update that PR later.